src/plugin.py

```diff
@@ -282,7 +282,7 @@ def create_total_games_cache(self):
         return int(datetime.datetime(year, month, day, hour, minute, second).timestamp())
 
     async def get_friends(self):
-        # NOTE: This will return a list of type FriendInfo.
+        # NOTE: This will return a list of type UserInfo.
```

Comment:
BTW
you can use python typehints:
`async def get_friends(self) -> typing.List[UserInfo]`

Reply:
I really should start using those. I might eventually get around to implementing typehints along with variable type declarations in method parameters for all existing methods, as it seems to be a nice habit to get into.

Reply:
its optional, we don't use it in most of plugins. But if you need comments for yourself like here, common practice in python is to use either typehints or docstings (there is a few standards you can search for)